samples/generated-code/src/test/java/com/yelp/codegen/generatecodesamples/InvokeRetrofitTest.kt

```diff
@@ -0,0 +1,26 @@
+package com.yelp.codegen.generatecodesamples
+
+import com.yelp.codegen.generatecodesamples.apis.DefaultApi
+import com.yelp.codegen.generatecodesamples.tools.GeneratedCodeConverters
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+
+class InvokeRetrofitTest {
+
+    @Test
+    fun createAndInvokeRetrofit() {
+        val retrofit: Retrofit = Retrofit.Builder()
+                .addCallAdapterFactory(RxJava2CallAdapterFactory.create())
+                .addConverterFactory(GeneratedCodeConverters.converterFactory())
+                .baseUrl("https://petstore.swagger.io/v2/")
```

Comment:
As far as I might like this idea, what is going to happen if `petstore.swagger.io` is offline? I do expect that our tests will start to fail.
Additionally this type of dependency forces developers to have internet connection while running tests which is not necessarily great.

My proposal would be to have a tiny mock server that catches requests and return "pre-defined" answers.
NOTE: I'm not looking for a complex system that is able to filter based on HTTP verb, path, parameters, etc. but rather something that "if I'm called then no mater what the answer is X" 😄